packages/procaptcha-react/src/components/ExtensionAccountSelect.tsx

```diff
@@ -44,25 +40,27 @@ export const ExtensionAccountSelect = ({
     const account: InjectedAccountWithMeta | null = accounts.find((a) => a.address === value) || null
 
     return (
-        <Autocomplete
-            disablePortal
+        // react select box
+        <select
             id="select-account"
-            options={accounts}
-            value={account}
-            isOptionEqualToValue={(option, value) => option.address === value.address}
-            onChange={(event: SyntheticEvent<Element, Event>, value: InjectedAccountWithMeta | null) => {
-                if (value) {
+            onChange={(e) => {
+                const value = e.target.value
+                const account = accounts.find((a) => a.address === value) || null
+                if (account) {
                     console.log('Selected account:', value)
-                    onChange(value.address)
+                    onChange(account.address)
                 } else {
                     console.log('Deselected account')
                     onChange('')
                 }
             }}
-            sx={{ width: 550 }}
-            getOptionLabel={(option: InjectedAccountWithMeta) => `${option.meta.name}: ${option.address}`}
-            renderInput={(props) => <TextField {...props} label={t('WIDGET.SELECT_ACCOUNT')} />}
-        />
+            value={accounts.length > 0 && account ? account.toString() : undefined}
```

Comment:
should this be account.address not account.toString()?

Reply:
yes, good point!